app/views/placements/wizards/add_placement_wizard/_check_your_answers_step.html.erb

```diff
@@ -57,6 +57,17 @@
           <% end %>
         <% end %>
 
+        <% if @wizard.steps.include?(:academic_year) %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: t(".academic_year")) %>
+            <% row.with_value(text: @wizard.steps[:academic_year].academic_year_display_name(@wizard.steps[:academic_year].academic_year)) %>
```

Comment:
Doesn't the step already know the academic year? So you don't need to pass it in as an argument.